modules/destroy-hc-ibmz-cli.adoc

```diff
@@ -20,8 +20,6 @@ $ oc -n <hosted_cluster_namespace> scale nodepool <nodepool_name> --replicas 0
 After the `NodePool` is scaled to `0`, the compute nodes get detached from the hosted cluster. In {product-title} version 4.17, this function is applicable only for {ibm-z-title} with KVM. For z/VM and LPAR we must delete the compute nodes manually.
 +
 If you want to re-attach compute nodes to the cluster, you can scale down a nodepool with desired number of compute nodes. Unlike {ibm-z-title} with KVM, in case of z/VM and LPAR  to reuse the agents ,you must recreate them by using the Discovery image.
```

Comment:
@krishnanshruthi 
a small correction here , 

If you want to re-attach compute nodes to the cluster, you can **scale out the nodepool** with desired number of compute nodes. Unlike {ibm-z-title} with KVM, in case of z/VM and LPAR  to reuse the agents ,you must recreate them by using the Discovery image.

Reply:
Done